docs/release/trg-0/trg-0.md

```diff
@@ -0,0 +1,18 @@
+---
+title: TRG 0 - Changelog & Drafts
```

Comment:
Just a small technicality remark: 

- I wouldn't call this folder/category TRG 0, but rather just `Drafts & Changelog`. 
- Then name the files in the folder like trg-5-9-helm-test.md as they would be named when they move out of draft to the released guides.